packages/cli/test/commands/pull.test.ts

```diff
@@ -19,60 +19,12 @@ describe('pull', () => {
     });
     client.setArgv('pull', '--yes', cwd);
     const exitCode = await pull(client);
-    expect(exitCode).toEqual(0);
+    expect(exitCode, client.outputBuffer).toEqual(0);
```

Comment:
Can you pass 2 params to expect() ? 🤔

Reply:
Oh I see your other comment now 😄

Reply:
Only with: https://github.com/vercel/vercel/pull/7656#discussion_r848607417